NanoXLSX/LowLevel/StyleReader.cs

```diff
@@ -451,7 +451,7 @@ private void GetCellXfs(XmlNode node)
                         // Undefined values may occur if the file was saved by an Excel version that has implemented yet unknown format numbers (undefined in NanoXLSX) 
                         format = new NumberFormat();
                         format.Number = formatNumber;
-                        format.InternalID = StyleReaderContainer.GetNextNumberFormatId();
+                        format.InternalID = id;

```

Comment:
In this case, we can do some cleanup, since the function GetNextNumberFormatId() is nowhere else use.
Please remove the function from the StyleContainer class.